src/pages/ActivationScreen.tsx

```diff
@@ -16,36 +16,47 @@ const InsertCardImage = styled.img`
 interface Props {
   election: Election
   isLiveMode: boolean
+  isVoterCardExpired: boolean
   isVoterCardInvalid: boolean
 }
 
 const ActivationScreen = ({
   election,
   isLiveMode,
+  isVoterCardExpired,
   isVoterCardInvalid,
 }: Props) => {
   const { title, state, county, date, seal, sealURL } = election
+  const showInvalid = isVoterCardInvalid
+  const showExpired = !showInvalid && isVoterCardExpired
+  const showDefault = !showInvalid && !showExpired
   return (
     <Main>
       <MainChild center>
         <Prose textCenter>
           <TestMode isLiveMode={isLiveMode} />
-          <Seal seal={seal} sealURL={sealURL} />
-          <h1 aria-label={`${title}.`}>{title}</h1>
-          <p aria-hidden="true">
-            {date}
-            <br />
-            {county.name}, {state}
-          </p>
-          <hr />
-          {isVoterCardInvalid ? (
+          {showInvalid && (
             <React.Fragment>
               <h1>Inactive Card</h1>
-              <p>This card is no longer active.</p>
-              <p>Please return card to poll worker.</p>
+              <p>Please return this card to a poll worker.</p>
             </React.Fragment>
-          ) : (
+          )}
+          {showExpired && (
             <React.Fragment>
+              <h1>Expired Card</h1>
+              <p>Please return this card to a poll worker for assistance.</p>
```

Comment:
This copy differs from the "Inactive Card" case. Should it?

Reply:
Good question. Yeah, I think something can be simplified here. 

There are two different states which perhaps we do not need to differentiate between for voters/poll workers. The resolution is the same for the Poll Worker, they simply re-encode the card. Though we could update the VxEncode UI to give the Poll Worker more information about the state of the card.

### Inactive card
- A voter card which has printed a ballot.
- If found and inserted into a BMD, the UI should display message that this card should be returned to the poll worker to reenter circulation.
- I'm trying to avoid using the copy "Used card" such that the card is tossed in the bin.

### Expired card
- A voter card which has not been used to print a ballot, but for which the voter too more than 10 minutes to insert into a BMD after being encoded.
- A voter should be able to re-activate this card with the help of a Poll Worker.

Any further thoughts based upon this info?

Reply:
Thanks for the context. I think this is fine as a first pass. We can always iterate.